core/jazz_services/config/dev-config.json

```diff
@@ -3,7 +3,8 @@
     "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status", "deployment_targets"],
     "service_update_fields": ["description", "email", "slack_channel", "tags", "repository", "status","metadata","is_public_endpoint"],
     "service_filter_params": ["service", "domain", "region", "type", "runtime", "created_by", "status"],
-    "service_optional_fields": ["description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint"],
+    "service_optional_fields": ["description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint", "deployment_descriptor"],
+    "service_return_fields": ["service", "domain", "type", "created_by", "runtime", "description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint"],
```

Comment:
Might have creeped in during merge conflict resolution - `service_return_fields` is a duplicate now, please remove it.

Reply:
Surprisingly one line contains fewer fields than the other. I have selected the line with largest number of fields

Reply:
Resolved with https://github.com/tmobile/jazz/pull/829/commits/0af8f0b94a3ac56f5d2d813795b2ce99cf092e41